setup.cfg

```diff
@@ -0,0 +1,7 @@
+[versioneer]
+VCS = git
+style = pep440-branch-based
+versionfile_source = nc_time_axis/_version.py
+versionfile_build = nc_time_axis/_version.py
+tag_prefix = v
+parentdir_prefix = nc-time-axis-
```

Comment:
No? It's not strictly necessary but I guess it would help people that do this sort of thing with a lot of repos, so they don't end up having a bunch of master.zip files